static/usage/v7/animations/keyframes/demo.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Keyframe Animations</title>
+    <link rel="stylesheet" href="../../../common.css" />
+    <script src="../../../common.js"></script>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@7/dist/ionic/ionic.esm.js"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@7/css/ionic.bundle.css" />
+    <script type="module">
+      import { createAnimation } from 'https://cdn.jsdelivr.net/npm/@ionic/core@7/dist/ionic/index.esm.js';
+
+      const animation = createAnimation()
+        .addElement(document.querySelector('.square'))
+        .duration(3000)
+        .iterations(Infinity)
+        .keyframes([
+          { offset: 0, background: 'red' },
+          { offset: 0.72, background: 'var(--background)' },
+          { offset: 1, background: 'aqua' },
+        ]);
+
+      document.querySelector('#play').addEventListener('click', () => {
+        animation.play();
+      });
+
+      document.querySelector('#pause').addEventListener('click', () => {
+        animation.pause();
+      });
+
+      document.querySelector('#stop').addEventListener('click', () => {
+        animation.stop();
+      });
+    </script>
+
+    <style>
+      .container {
+        flex-direction: column;
+      }
+
+      .square {
+        width: 100px;
+        height: 100px;
+        background: red;
+        --background: yellow;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="square"></div>
```

Comment:
Could we use an `ion-card` here like the other demos? Example: https://ionicframework.com/docs/utilities/animations#basic-animations